animation_nodes/nodes/sound/MIDI_Note_Evaluate.py

```diff
@@ -0,0 +1,65 @@
+import bpy
+from ... base_types import AnimationNode, ListTypeSelectorSocket, VectorizedSocket
+from ... data_structures import DoubleList
```

Comment:
Order imports based on line length.

Reply:
Enlight me on this one too please

Reply:
Switch the second and third lines such that shorter lines are at the top.